torch/_inductor/codegen/cuda/cutlass_epilogue_gen.py

```diff
@@ -0,0 +1,202 @@
+from typing import cast, List
+from unittest.mock import patch
+
+import sympy
+
+import torch._inductor.virtualized as virtualized
+from torch._inductor.ir import ComputedBuffer, FlexibleLayout, IRNode, Pointwise
+from torch._inductor.utils import IndentedBuffer, sympy_str
+
+
+def _arg_str(a):
+    if isinstance(a, sympy.Expr):
+        return "sympy_expr('" + sympy_str(a) + "')"
+    return str(a)
+
+
+class CutlassEVTEpilogueTypeFormatter:
+    """
+    Replacement for V.KernelFormatterHandler
+    """
+
+    def __init__(self, accumulator_node_name, evt_type_name):
+        self.accumulator_node_name = accumulator_node_name
+        self.output = IndentedBuffer(0)
+        self.var_counter = 0
+        self.evt_type_name = evt_type_name
+        self.aliases = dict()
+
+    @staticmethod
+    def ir_to_evt_string(
+        template_output_node_name: str,
+        evt_type_name: str,
+        epilogue_nodes: List[IRNode],
+    ):
+        formatter = CutlassEVTEpilogueTypeFormatter(
```

Comment:
The instance needs to know how the output node of the matmul is called, because it it encounters a "load" instruction with that name, it's translated to a "Load Accumulator" op.